Utilities/ReleaseNoteParameters.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ReleaseNotes_WebAPI.Utilities
+{
+    // https://code-maze.com/filtering-aspnet-core-webapi/
+    public class ReleaseNoteParameters
+    {
+        public int product { get; set; }
+        // TODO: remove if unnec
+        public string sort { get; set; } = "";
+    }
+}
```

Comment:
Ka brukes den her tell?

Reply:
den ble tidliger brukt i  et forsøk på routing for å få spesifikke release notes. Endte opp med en annen løsning så denne fjernes.